core/src/test/scala/org/ensime/config/EnsimeConfigSpec.scala

```diff
@@ -26,27 +21,26 @@ class EnsimeConfigSpec extends FunSpec with Matchers {
 
     it("should parse a simple config") {
       withCanonTempDir { dir =>
-        (dir / ".ensime_cache").mkdirs()
-        (dir / "abc").mkdirs()
+        val abc = dir / "abc"
+        val cache = dir / ".ensime_cache"
+        val javaHome = file(Properties.javaHome)
 
-        val dirStr = fileToWireString(dir)
-        val abc = fileToWireString(dir / "abc")
-        val cacheStr = fileToWireString(dir / ".ensime_cache")
-        val javaHome = fileToWireString(file(Properties.javaHome))
+        abc.mkdirs()
+        cache.mkdirs()
 
         test(dir, s"""
 (:name "project"
  :scala-version "2.10.4"
- :java-home $javaHome
- :root-dir $dirStr
- :cache-dir $cacheStr
+ :java-home "$javaHome"
```

Comment:
This won't work, $javaHome etc. contain backslashes which break the lisp parser. They really must be escaped.